dashboard/app/models/script_level.rb

```diff
@@ -327,7 +327,8 @@ def summarize(include_prev_next=true, for_edit: false, user_id: nil)
         url: build_script_level_url(self),
         freePlay: level.try(:free_play) == "true",
         bonus: bonus,
-        display_as_unplugged: level.display_as_unplugged?
+        display_as_unplugged: level.display_as_unplugged?,
+        app: level.game.app
```

Comment:
could a level ever not have a game?

Reply:
I'm not sure.  I haven't come across it yet, but am keeping an eye out.

Reply:
If we aren't sure, it is probably worth doing a null check here, given this method is used widely.

Reply:
Good point.  I'm hesitant about too many `&.`s in case they mask bad assumptions, and we do refer to `level.game` elsewhere in this file and other files.

From querying the production database, of 45087 levels, 2 are missing `game`, and of the remaining levels, 89 are missing `app`.  It feels unlikely we'll encounter them in a script, but I'll make it `level.game&.app` to be extra safe.